### PR TITLE
Updated API Document.

### PR DESCRIPTION
**Pull request by cenkiravul**
Updated API Document and examples.

### CODE REVIEW

File: content/apidocs-mxsdk/apidocs/deploy-api.md [hunk 1]
```diff
@@ -218,29 +218,30 @@ List of objects with the following key-value pairs:
 
 ##### <a rel="nofollow"></a>Example
 
-```java
-[{
+```bash
```
Comment (pommi) [inline content/apidocs-mxsdk/apidocs/deploy-api.md]: At line 174 it's also ` ```java ` and you didn't change it there. Should it be ` ```bash ` or ` ```java `?

File: content/apidocs-mxsdk/apidocs/deploy-api.md [hunk 2]
```diff
@@ -218,29 +218,30 @@ List of objects with the following key-value pairs:
...
+	},
+	{
      "Status" :  "Stopped" ,
+     "Mode" :  "Production",
      "Url" :  "https://calc.mendixcloud.com"
-                 ,
-     "Mode" :  "Production"
-}]
+	}
+]
```
Comment (pommi) [inline content/apidocs-mxsdk/apidocs/deploy-api.md]: I think you are mixing up tabs and spaces in the above code block.

File: content/apidocs-mxsdk/apidocs/deploy-api.md [hunk 3]
```diff
@@ -281,17 +282,16 @@ An object with the following key-value pairs:
 ```bash
 {
      "Status" :  "Stopped" ,
-     "Url" :  "https://calc-accp.mendixcloud.com"
-                 ,
      "Mode" :  "Acceptance"
```
Comment (pommi) [inline content/apidocs-mxsdk/apidocs/deploy-api.md]: There is a comma missing at the end of this line.

File: content/apidocs-mxsdk/apidocs/deploy-api.md [hunk 4]
```diff
@@ -304,7 +304,7 @@ URL: [https://deploy.mendix.com/api/1/apps/<AppId>/environments/<Mode>/start (ht
 
 An object with the following key-value pair:
 
-*   _AutoSyncDb_ (Boolean): Define whether the database should be synchronized automatically with the model during the start phase of the app.
+*   _AutoSyncDb_ (Boolean) : Define whether the database should be synchronized automatically with the model during the start phase of the app. This is only applicable if your Mendix Cloud version is older than v4.
```
Comment (pommi) [inline content/apidocs-mxsdk/apidocs/deploy-api.md]: "v4 and older", not "older than v4", that means v5 and up.

File: content/apidocs-mxsdk/apidocs/deploy-api.md (hunk 1 above)
Comment (pommi) [inline content/apidocs-mxsdk/apidocs/deploy-api.md]: Hmm, looking further, it's a whole mixture of ` ```bash ` and ` ```java `. I'm not sure what to do now.

File: content/apidocs-mxsdk/apidocs/deploy-api.md [hunk 5]
```diff
@@ -602,11 +603,11 @@ Mendix-ApiKey:  26587896-1cef-4483-accf-ad304e2673d6
 
 ### <a rel="nofollow"></a>Description
 
-Removes all data from a specific environment including files and database records. This action requires the environment to be in "NotRunning" status. 
+Removes all data from a specific environment including files and database records. This action requires the environment to be in "NotRunning" status.
 
 ```bash
 HTTP Method: POST
-URL: [https://deploy.mendix.com/api/1/apps/<AppId>/environments/<Mode>/clean (https://deploy.mendix.com/api/1/apps/<AppId>/environments/<Mode>/clean)]
+URL: [https://deploy.mendix.com/api/1/apps/<AppId>/environments/<Mode>/clean
```
Comment (pommi) [inline content/apidocs-mxsdk/apidocs/deploy-api.md]: You forgot to remove the `[`

File: content/apidocs-mxsdk/apidocs/deploy-api.md [hunk 6]
```diff
@@ -629,6 +630,18 @@ Mendix-ApiKey:  26587896-1cef-4483-accf-ad304e2673d6
...
+
+```bash
+[
+	{
+    "Status": "Stopped",
+    "Mode": "Acceptance",
+    "Url" : "https://calc-accp.mendixcloud.com"
+	}
+]
+```
```
Comment (pommi) [inline content/apidocs-mxsdk/apidocs/deploy-api.md]: I think you are mixing up tabs and spaces in the above code block.

File: content/apidocs-mxsdk/apidocs/deploy-api.md [hunk 7]
```diff
@@ -643,11 +656,11 @@ Mendix-ApiKey:  26587896-1cef-4483-accf-ad304e2673d6
 
 ### <a rel="nofollow"></a>Description
 
-Gets current values of custom settings, constants and scheduled events used by the target environment. 
+Gets current values of custom settings, constants and scheduled events used by the target environment.
 
 ```bash
 HTTP Method: GET
-URL: [https://deploy.mendix.com/api/1/apps/<AppId>/environments/<Mode>/settings/ (https://deploy.mendix.com/api/1/apps/<AppId>/environments/<Mode>/settings/)]
+URL: [https://deploy.mendix.com/api/1/apps/<AppId>/environments/<Mode>/settings
```
Comment (pommi) [inline content/apidocs-mxsdk/apidocs/deploy-api.md]: You forgot to remove the `[`

File: content/apidocs-mxsdk/apidocs/deploy-api.md [hunk 8]
```diff
@@ -706,11 +719,11 @@ Mendix-ApiKey:  26587896-1cef-4483-accf-ad304e2673d6
 
 ### <a rel="nofollow"></a>Description
 
-Changes value of existing environment settings like custom settings, constants and scheduled events. These changes are applied after restarting the environment. 
+Changes value of existing environment settings like custom settings, constants and scheduled events. These changes are applied after restarting the environment.
 
 ```bash
 HTTP Method: POST
-URL: [https://deploy.mendix.com/api/1/apps/<AppId>/environments/<Mode>/settings/ (https://deploy.mendix.com/api/1/apps/<AppId>/environments/<Mode>/settings/)]
+URL: [https://deploy.mendix.com/api/1/apps/<AppId>/environments/<Mode>/settings/
```
Comment (pommi) [inline content/apidocs-mxsdk/apidocs/deploy-api.md]: You forgot to remove the `[`

File: content/apidocs-mxsdk/apidocs/deploy-api.md (hunk 1 above)
Comment (cenkiravul) [inline content/apidocs-mxsdk/apidocs/deploy-api.md]: Changed all to ```bash for consistency.

File: content/apidocs-mxsdk/apidocs/deploy-api.md (hunk 2 above)
Comment (cenkiravul) [inline content/apidocs-mxsdk/apidocs/deploy-api.md]: Fixed.

File: content/apidocs-mxsdk/apidocs/deploy-api.md (hunk 1 above)
Comment (cenkiravul) [inline content/apidocs-mxsdk/apidocs/deploy-api.md]: Changed all to ```bash for consistency

File: content/apidocs-mxsdk/apidocs/deploy-api.md (hunk 3 above)
Comment (cenkiravul) [inline content/apidocs-mxsdk/apidocs/deploy-api.md]: Fixed.

File: content/apidocs-mxsdk/apidocs/deploy-api.md (hunk 6 above)
Comment (cenkiravul) [inline content/apidocs-mxsdk/apidocs/deploy-api.md]: Fixed.

File: content/apidocs-mxsdk/apidocs/deploy-api.md (hunk 4 above)
Comment (cenkiravul) [inline content/apidocs-mxsdk/apidocs/deploy-api.md]: Mutually agreed to leave it as it is.

File: content/apidocs-mxsdk/apidocs/deploy-api.md (hunk 5 above)
Comment (cenkiravul) [inline content/apidocs-mxsdk/apidocs/deploy-api.md]: Fixed.

File: content/apidocs-mxsdk/apidocs/deploy-api.md (hunk 7 above)
Comment (cenkiravul) [inline content/apidocs-mxsdk/apidocs/deploy-api.md]: Fixed.

File: content/apidocs-mxsdk/apidocs/deploy-api.md (hunk 8 above)
Comment (cenkiravul) [inline content/apidocs-mxsdk/apidocs/deploy-api.md]: Fixed.

**Pull request merged**